Code/Framework/AzFramework/Platform/Mac/platform_nativeui_mac.cmake

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+find_library(APPKIT_LIBRARY AppKit)
+find_library(GAME_CONTROLLER_LIBRARY GameController)
+find_library(CARBON_LIBRARY Carbon)
```

Comment:
Do we use this? if not we should probably remove it if it's the same as `Carbon Core` which is deprecated https://developer.apple.com/documentation/coreservices/carbon_core?language=objc
Can be a future PR.

Reply:
Yeah, this was a move to a new file, and removing it should be a different PR
Created a GHI for this: https://github.com/o3de/o3de/issues/16644